src/wasm-lib/grackle/src/kcl_value_group.rs

```diff
@@ -59,7 +60,8 @@ impl From<ast::types::Value> for KclValueGroup {
             ast::types::Value::ArrayExpression(e) => Self::ArrayExpression(e),
             ast::types::Value::ObjectExpression(e) => Self::ObjectExpression(e),
             ast::types::Value::MemberExpression(e) => Self::Single(SingleValue::MemberExpression(e)),
-            ast::types::Value::PipeSubstitution(_) | ast::types::Value::FunctionExpression(_) => todo!(),
+            ast::types::Value::FunctionExpression(e) => Self::Single(SingleValue::FunctionExpression(e)),
+            ast::types::Value::PipeSubstitution(_) => todo!(),
```

Comment:
I think technically in the future your function support can remove pipe operator in favor of a KCL defined pipe operator, right?

Reply:
Hmmm. Maybe... haven't thought about it too much.